app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+
+get '/tweets' do
+    if session[:id]
```

Comment:
You can use your `logged_in?` helper method here, which is preferable to referring to the session object directly.